libres/lib/enkf/enkf_analysis.cpp

```diff
@@ -27,164 +27,188 @@
+    if (local_inactive_obs == true)
+        fprintf(stream, "* Local inactive\n");
+    fprintf(stream, "\n\n\n");
 }
 
-
-
-
-void enkf_analysis_deactivate_outliers(obs_data_type * obs_data , meas_data_type * meas_data , double std_cutoff , double alpha, bool verbose) {
-  for (int block_nr =0; block_nr < obs_data_get_num_blocks( obs_data ); block_nr++) {
-    obs_block_type  * obs_block  = obs_data_iget_block( obs_data , block_nr);
-    meas_block_type * meas_block = meas_data_iget_block( meas_data , block_nr );
-
-    {
-      int iobs;
-      for (iobs =0; iobs < meas_block_get_total_obs_size( meas_block ); iobs++) {
-        if (meas_block_iget_active( meas_block , iobs )) {
-          double ens_std  = meas_block_iget_ens_std( meas_block , iobs );
-          if (ens_std <= std_cutoff) {
-            /*
+void enkf_analysis_deactivate_outliers(obs_data_type *obs_data,
+                                       meas_data_type *meas_data,
+                                       double std_cutoff, double alpha,
+                                       bool verbose) {
+    for (int block_nr = 0; block_nr < obs_data_get_num_blocks(obs_data);
+         block_nr++) {
+        obs_block_type *obs_block = obs_data_iget_block(obs_data, block_nr);
+        meas_block_type *meas_block = meas_data_iget_block(meas_data, block_nr);
+
+        {
+            int iobs;
+            for (iobs = 0; iobs < meas_block_get_total_obs_size(meas_block);
+                 iobs++) {
+                if (meas_block_iget_active(meas_block, iobs)) {
+                    double ens_std = meas_block_iget_ens_std(meas_block, iobs);
+                    if (ens_std <= std_cutoff) {
+                        /*
               De activated because the ensemble has to small
               variation for this particular measurement.
             */
-            obs_block_deactivate( obs_block , iobs , verbose , "No ensemble variation");
-            meas_block_deactivate( meas_block , iobs );
-          } else {
-            double ens_mean  = meas_block_iget_ens_mean( meas_block , iobs );
-            double obs_std   = obs_block_iget_std( obs_block , iobs );
-            double obs_value = obs_block_iget_value( obs_block , iobs );
-            double innov     = obs_value - ens_mean;
-
-            /*
+                        obs_block_deactivate(obs_block, iobs, verbose,
+                                             "No ensemble variation");
+                        meas_block_deactivate(meas_block, iobs);
+                    } else {
+                        double ens_mean =
+                            meas_block_iget_ens_mean(meas_block, iobs);
+                        double obs_std = obs_block_iget_std(obs_block, iobs);
+                        double obs_value =
+                            obs_block_iget_value(obs_block, iobs);
+                        double innov = obs_value - ens_mean;
+
+                        /*
```

Comment:
Indentation of comment now looks weird because the indentation of the code has changed.